.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: '>=1.20.0'
+          go-version: '>=1.21.0'
```

Comment:
I'd make this 1.21 exactly instead of >=1.21, so that when 1.22 comes out, we are still consistent between build scripts and CI.

Reply:
Done